marklogic-client-api-functionaltests/src/test/java/com/marklogic/client/fastfunctest/AbstractFunctionalTest.java

```diff
@@ -57,7 +57,7 @@ public static void initializeClients() throws Exception {
         isML11OrHigher = version.getMajor() >= 11;
 
 		client = newDatabaseClientBuilder().build();
-		schemasClient = newClientForDatabase("java-functest-schemas");
```

Comment:
`newClientForDatabase` was only used in 2 places, which doesn't seem like justification enough for a helper method since it barely saves on any code.

Reply:
Would it be useful to the users of the client?

Reply:
`newClientForDatabase` was in src/test/java. I'm trying to slim down `ConnectedRestQA` as much as possible so it's easier to comprehend. So if there's a helper method only used in one or two places that doesn't help much at all, I'm removing those. No impact on Java Client users.